addons/website_sale/data/data.xml

```diff
@@ -22,6 +22,12 @@
             <field name="name">Brand</field>
             <field name="sequence">0</field>
         </record>
+        <record id="product_barcodelookup.product_attribute_5" model="product.attribute">
+            <field name="visibility">hidden</field>
+        </record>
+        <record id="product_barcodelookup.product_attribute_6" model="product.attribute">
+            <field name="visibility">hidden</field>
+        </record>
```

Comment:
Sadly, you cannot reference a record which is not defined in the module dependencies (does it really matter to hide those though) ?

Reply:
I have changed the code accordingly now we will create these attributes when encountered but not available.